docs/semgrep-ci/ci-environment-variables.md

```diff
@@ -186,7 +186,7 @@ pipelines:
 
 ### `SEMGREP_REPO_NAME`
 
-Set `SEMGREP_REPO_NAME` to define the repository name for the URL used to generate hyperlinks in the [Findings](/docs/semgrep-code/findings) page. To avoid hardcoding this value, check your CI provider's documentation for available environment variables that can automatically detect the correct values for every CI job. 
```

Comment:
A recent change to how we construct hyperlinks on the findings page allows us to read only from `SEMGREP_REPO_URL` to construct the hyperlinks. Now, the `SEMGREP_REPO_NAME` is used to display the repo name on the app along with construction of API URL paths to make calls to external SCM APIs to publish review comments on PRs/MRs.

Reply:
👍